commercetools-models/src/main/java/io/sphere/sdk/productdiscounts/queries/MatchingProductDiscountGet.java

```diff
@@ -0,0 +1,39 @@
+package io.sphere.sdk.productdiscounts.queries;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.sphere.sdk.client.SphereRequest;
+import io.sphere.sdk.productdiscounts.ProductDiscount;
+import io.sphere.sdk.products.Price;
+
+/**
+ * This endpoint can be used to simulate which product discounts would be applied if a specified product variant had a specified price.<br>
+ * Given product and variant ids and a price, this endpoint will return the product discount that would have been applied to that price.
+ */
+public interface MatchingProductDiscountGet extends SphereRequest<ProductDiscount> {
+
+    /**
+     * @return UUID, ID of the product
+     */
+    @JsonProperty
+    String getProductId();
```

Comment:
I Don't know how useful it is to provide access to the properties of a request here. I would suggest to hide these properties in the implementation so that this interface is more consistent with our existing interfaces.

Reply:
👍 makes sense